### PR TITLE
Feat/[T82-440] Sweep modal

### PR DESCRIPTION
**Pull request by JoaoMarcelo-J**

### CODE REVIEW

File: src/hooks/useCoinConversion.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      revalidateOnFocus: false,
+      refreshWhenHidden: false,
+    }
+  )
+
+  if (data && data[0] && data[0].current_price) {
+    return data[0].current_price as number
+  }
+  return null
+}
```
Comment (luanpontolio) [inline src/hooks/useCoinConversion.tsx:24]: Necessary empty line in the end of this file

File: src/hooks/useTokens.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+function setParams(url: string | URL, query: { [x: string]: any }) {
+  if (typeof url === 'string') {
+    const searchParams = new URLSearchParams(query)
+    return `${url}?${searchParams.toString()}`
+  }
+  Object.keys(query).map((key) =>
+    url.searchParams.set(key, query[key]?.toString())
+  )
+  return url.href
+}
```
Comment (luanpontolio) [inline src/hooks/useTokens.tsx:58]: Same problem

**Review by freitasgouvea — APPROVED** [review]

File: src/hooks/useCoinConversion.tsx (hunk 1 above)
Comment (JoaoMarcelo-J) [reply src/hooks/useCoinConversion.tsx:24]: this line there isnt in my local

File: src/hooks/useTokens.tsx (hunk 2 above)
Comment (JoaoMarcelo-J) [reply src/hooks/useTokens.tsx:58]: here there isnt too

**Review by luanpontolio — APPROVED** [review]
🎩

**Pull request merged**